src/wizard/wizard.scss

```diff
@@ -60,110 +59,79 @@
-        }
+    .iui-step-name {
+      text-align: center;
+      user-select: all;
+      @include themed {
+        color: t(iui-text-color-muted);
       }
     }
 
-    > .iui-wizards-step-current {
-      > .iui-wizards-step-title {
+    &.iui-current {
+      .iui-step-name {
         @include themed {
           color: t(iui-color-foreground-positive);
         }
         font-weight: $iui-font-weight-semibold;
       }
 
-      > .iui-wizards-step-circle {
+      .iui-circle {
+        font-weight: $iui-font-weight-semibold;
         @include themed {
           color: t(iui-color-foreground-accessory);
-          border: 1px solid t(iui-color-background-positive);
+          border-color: t(iui-color-background-positive);
           background-color: t(iui-color-background-positive);
         }
-        font-weight: $iui-font-weight-semibold;
       }
 
-      > .iui-wizards-step-track-before {
+      .iui-track:first-child {
         @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
+          background-color: t(iui-color-foreground-positive);
         }
       }
     }
 
-    > .iui-wizards-step-completed {
-      &.iui-clickable {
-        > .iui-wizards-step-circle {
-          @include iui-focus(iui-focus-positive-box-shadow);
-          cursor: pointer;
-          @media (prefers-reduced-motion: no-preference) {
-            transition: background-color $iui-speed-fast ease-out;
-          }
+    &.iui-completed {
+      &.iui-clickable .iui-circle {
+        @include iui-focus(iui-focus-positive-box-shadow);
 
-          &:hover {
-            @include themed {
-              color: t(iui-color-foreground-accessory);
-              border: 1px solid t(iui-color-background-positive);
-              background-color: t(iui-color-background-positive);
-            }
+        &:hover {
+          cursor: pointer;
+          @include themed {
+            background-color: t(iui-color-background-positive);
+            border-color: t(iui-color-background-positive);
+            color: t(iui-color-foreground-accessory);
           }
         }
       }
 
-      > .iui-wizards-step-title {
+      .iui-step-name {
         @include themed {
           color: t(iui-color-foreground-positive);
         }
       }
 
-      > .iui-wizards-step-circle {
+      .iui-circle {
         @include themed {
           color: t(iui-color-foreground-positive);
-          border: 1px solid t(iui-color-foreground-positive);
+          border-color: t(iui-color-foreground-positive);
         }
       }
 
-      > .iui-wizards-step-track {
+      .iui-track {
         @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
+          background-color: t(iui-color-foreground-positive);
         }
       }
     }
+
+    &:first-child .iui-track:first-child,
+    &:last-child .iui-track:last-child {
+      background-color: transparent;
+    }
+  }
+
+  .iui-wizard-steps {
```

Comment:
I don't think this one is needed, maybe just don't add steps where they are not needed.

Reply:
If I remove this line of CSS and keep the HTML structure the same between the default and long wizard, the default wizard will look like this:
![Screen Shot 2021-08-05 at 1 21 01 PM](https://user-images.githubusercontent.com/849817/128393804-f1f4a8c3-f93f-413f-8d2d-df0c80b54c4c.png)

If you'd rather handle showing / hiding that portion in React I can remove this.

Reply:
We are already doing that now in React.

Reply:
Ok, I have removed this part.  Should I also remove line 6-8 within `long.scss` as it's doing something similar?

Reply:
Yeah, remove from there too.

Reply:
Ok, removing in the next push.